src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/service/LocationService.kt

```diff
@@ -212,6 +212,10 @@ class LocationService(
 
     residentialLocation.update(patchLocationRequest, authenticationFacade.getUserOrSystemInContext(), clock)
 
+    //if request has usedFor then we need to cascade usedfor
+    if(patchLocationRequest.usedFor?.isNotEmpty() == true)
```

Comment:
So this could work, but I think there needs to be an explicit endpoint which indicate that we want to cascade this down.  I don't think we should do this in a PATCH

Reply:
gotcha. that was my first approach, i will create a Endpoint that is straightfoward for me. no problem, i will update this afternoon.